src/printing.jl

```diff
@@ -225,15 +238,19 @@ function print_sourcecode(io::IO, code::String, line::Integer, defline::Integer)
         code[i] = code[i][min_indentation+1:end]
     end
 
-    stoplinelength += stoplinelength == ndigits(current_line)
+    filter!(x -> x in(startline:stopline), breakpoint_lines)
+
+    # If we print a token on a line with the same digits as the last line we need to add 1
+    stoplinelength += (ndigits(current_line) == stoplinelength) || (any(x -> ndigits(x) == stoplinelength, breakpoint_lines))
     for textline in code
-        prefix = lineno == current_line ? ">" : 
-                        ndigits(lineno) > ndigits(current_line) ? 
-                        "" :
-                        " "
+        break_on_line = lineno in breakpoint_lines
+        prefix = "", :normal
+        break_on_line           && (prefix = ('●', :light_red))
+        lineno == current_line  && (prefix = ('>', :yellow))
         printstyled(io,
-            string(lpad(string(prefix, lineno), stoplinelength , " "), "  "),
-            color = lineno == current_line ? :yellow : :bold)
+            string(lpad(string(prefix[1], lineno), stoplinelength , " "), "  "),
```

Comment:
Maybe
```
string(string(prefix[1], lpad(lineno, stoplinelength, " "), "  "),
```
instead? That way the breakpoints/current line indicators are consistently in one column (much like in an editor).

Reply:
I thought about that too but not sure it looked better:

![deb2](https://user-images.githubusercontent.com/1282691/54531972-e781ce80-4986-11e9-8c19-bbba30d30cdd.PNG)

Reply:
I like it better, fwiw ;)
Would also add a space between the symbols and the line numbers, but maybe people feel like their horizontal space is too precious for that?

Reply:
Updated to your suggestion :)